controllers/component_build_controller_test.go

```diff
@@ -1330,6 +1457,52 @@ var _ = Describe("Component initial build controller", func() {
 
 			ensureNoPipelineRunsCreated(resourceKey)
 		})
+
+		It("should not submit initial build if Dockerfile is wrong", func() {
+			DevfileSearchForDockerfile = func(devfileBytes []byte) (*v1alpha2.DockerfileImage, error) {
+				return nil, fmt.Errorf("wrong dockerfile")
+			}
+			setComponentDevfileModel(resourceKey)
+			ensureNoPipelineRunsCreated(resourceKey)
+			DevfileSearchForDockerfile = devfile.SearchForDockerfile
```

Comment:
This is fragile because the test might fail before reaching this line.

Reply:
ok moving to beforeEach

Reply:
ok